deploy/role.yaml

```diff
@@ -53,3 +53,9 @@ rules:
   - '*'
   verbs:
   - '*'
+- apiGroups:
+  - autoscaling/v2beta2
```

Comment:
This should be only `autoscaling`. I think that's the reason why integration tests are breaking.

Reply:
Makes sense, thank @cezarsa.